### PR TITLE
Add informations about the button height

### PR DESCRIPTION
**Pull request by cx384**
I have noticed that the button height can't actual be changed, like the field height. For fields this is documented but for buttons not. Therefore I made this pull request.

### CODE REVIEW

File: doc/lua_api.txt [hunk 1]
```diff
@@ -1857,6 +1857,7 @@ examples.
 * Clickable button. When clicked, fields will be sent.
 * `x`, `y` and `name` work as per field
 * `w` and `h` are the size of the button
+* Buttons are a set height, but will be vertically centred on `h`
```
Comment (SmallJoker) [inline doc/lua_api.txt]: How about `Fixed button height. It will be vertically centred on 'h'`?

**Comment by SmallJoker** [comment]
Consistent with the CSM docs, I see. A description like `Fixed button height. It will be vertically centred on 'h'` would sound better IMO. \*shrug*
:+1:

**Comment by cx384** [comment]
Yeah, thats better, but I tried to keep the lua_api.txt style.
https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L1818

**Comment by DS-Minetest** [comment]
Related: #6047

**Comment by SmallJoker** [comment]
@cx384 Just because it's there doesn't mean it's good ;)

**Comment by cx384** [comment]
Updated client_lua_api.md, too.

**Comment by nerzhul** [comment]
i tend to prefer @SmallJoker sentence

**Comment by SmallJoker** [comment]
Merged as d60434c

**Pull request closed without merging**